### DISCUSSION TITLE
Security Onion in the Oracle Cloud Infrastructure

### DISCUSSION THREAD

**Discussion by rsjr81** [Security-Onion-Solutions/securityonion#10061] [Unsupported Versions]
Hello,

I am trying to standup a security onion in the Oracle Cloud Infrastructure (OCI) and I have not been able to get it working yet.  Does anyone know if this has been implemented before?  And in general should I be able to get the SO to install/work on top of CentOS in cloud infrastructure or does a special image need to be created for AWS, Azure, OCI in order for it to work properly?

Thanks.

**Comment by InfosecGoon** [comment]
I don't believe we've done any testing in Oracle Cloud, but you should be able to do a network install on top of a CentOS 7 instance. Just make sure that all of the necessary domains and ports are open in your security configuration.

Network install: https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

Network requirements: https://docs.securityonion.net/en/2.3/firewall.html
